src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2506,9 +2507,6 @@ NetworkOPsImp::recvValidation(
     std::shared_ptr<STValidation> const& val,
     std::string const& source)
 {
-    JLOG(m_journal.debug())
-        << "recvValidation " << val->getLedgerHash() << " from " << source;
```

Comment:
I think this is useful for debugging, especially with validation/proposal reduce-relay feature enabled. Maybe just change it to `trace` to reduce the noise?

Reply:
:+1: